python/ray/tests/test_state_api.py

```diff
@@ -3559,6 +3559,28 @@ def test_core_state_api_usage_tags(shutdown_only):
     )
 
 
+def test_job_info_is_running_task(shutdown_only):
+    ray.init()
+
+    @ray.remote
+    def f():
+        import time
+
+        while True:
+            time.sleep(10000)
+
+    tasks = [f.remote() for i in range(4)]
+
+    client = ray.worker.global_worker.gcs_client
+    job_id = ray.worker.global_worker.current_job_id.binary()
+    all_job_info = client.get_all_job_info()
+    assert len(all_job_info) == 1
+    assert job_id in all_job_info
+    assert client.get_all_job_info()[job_id].is_running_tasks is True
+    for o in tasks:
+        ray.cancel(o)
```

Comment:
Do you need these, the cluster is terminating anyway

Reply:
if we don't use `tasks`, the linter complains. I changed the cancel to a print to mitigate.

Reply:
In second thought, I can add a noqa lint directive for the unused variables